src/status_im/contexts/communities/overview/events.cljs

```diff
@@ -120,10 +120,11 @@
   [_ [community-id addresses-to-reveal signatures]]
   {:fx [[:json-rpc/call
          [{:method      "wakuext_requestToJoinCommunity"
-           :params      [{:communityId       community-id
-                          :signatures        signatures
-                          :addressesToReveal addresses-to-reveal
-                          :airdropAddress    (first addresses-to-reveal)}]
+           :params      [{:communityId          community-id
+                          :signatures           signatures
+                          :addressesToReveal    addresses-to-reveal
+                          :shareFutureAddresses share-future-addresses
```

Comment:
`share-future-addresses` is undefined @vitvly.

Reply:
Fixed.